### PR TITLE
fix: Fatal error due to undeclared static color_group_configs var

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Fatal error caused due to undeclared static color_group_configs var
- Removed this code as it's not needed
- Control / sub-control relationship managed via customizer.js only

### Screenshots
- https://a.cl.ly/wbu9Ob5l

### Types of changes
- Non breaking change
- Removed extra unwanted code

### How has this been tested?
- With color group control in customizer
- Check whether all color-group controls are working properly

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/customizer/class-astra-customizer.php [hunk 1]
```diff
@@ -490,13 +490,6 @@ public static function bypass_control_configs( $configuration ) {
 					}
 					$configuration['ast_fields'] = $config;
 					break;
-				case 'ast-color-group':
```
Comment (rohitp-bsf) [inline inc/customizer/class-astra-customizer.php:493]: Do we need this control? `class-astra-control-color-group.php` same static property is used inside that? Please check @imnavanath

**Review by rohitp-bsf — APPROVED** [review]

File: inc/customizer/class-astra-customizer.php (hunk 1 above)
Comment (imnavanath) [reply inc/customizer/class-astra-customizer.php:493]: Right, the file is also not needed. So removed this class control file.

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,6 @@
+v3.1.2
+- Fix: Fatal error due to undeclated static property Astra_Customizer::$color_group_configs.
```
Comment (vrundakansara) [inline changelog.txt:2]: `- Fix: Fatal error due to undeclared static property Astra_Customizer::$color_group_configs.`

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: changelog.txt (hunk 2 above)
Comment (imnavanath) [reply changelog.txt]: Done thanks 👍

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**